check_stability.py

```diff
@@ -592,11 +637,18 @@ def main():
     """Perform check_stability functionality and return exit code."""
     global wpt_root
     global wptrunner_root
+    global logger
 
     retcode = 0
     parser = get_parser()
     args = parser.parse_args()
 
+    replace_streams(args.output_bytes,
```

Comment:
Looks like I can support this by simply not specifying a "default" value when defining the option.